core/domain/prod_validation_jobs_one_off.py

```diff
@@ -280,6 +283,9 @@ def _get_custom_validation_functions(cls):
 
         Each validation function should accept only a single arg, which is the
         model instance to validate.
+
+        Returns:
+            list(*): The list of custom validation functions to run.
```

Comment:
Actually, the convention is to use colons after arg names, but to use periods after exception names and typeinfos. (It's possible that some of the other docstrings in this file are wrong -- would it be possible to fix those accordingly?) See core/domain/change_domain.py for an example of the intended convention.

Ditto elsewhere.